client/src/pages/locations/GeoLocation.js

```diff
@@ -218,29 +211,36 @@ const LatLonFormField = ({
           <Field
             name="lat"
             component={FieldHelper.InputFieldNoLabel}
-            onBlur={() => {
-              setParsedLatLng(lat, lng)
+            onChange={e => setLatOnChange(e.target.value)}
+            onBlur={e => {
+              setParsedLatOnBlur(e.target.value)
             }}
           />
         </Col>
         <Col sm={3}>
           <Field
             name="lng"
             component={FieldHelper.InputFieldNoLabel}
-            onBlur={() => {
-              setParsedLatLng(lat, lng)
+            onChange={e => setLngOnChange(e.target.value)}
+            onBlur={e => {
+              setParsedLngOnBlur(e.target.value)
             }}
           />
         </Col>
         <CoordinateActionButtons
-          lat={lat}
-          lng={lng}
+          coordinates={coordinates}
           isSubmitting={isSubmitting}
-          disabled={!lat && lat !== 0 && !lng && lng !== 0}
+          disabled={
+            !coordinates.lat &&
```

Comment:
You can use here the variables you defined above (lt/ln , but probably you will rename them to lat/lng) instead.
So actually the change of the disabled value is not needed.